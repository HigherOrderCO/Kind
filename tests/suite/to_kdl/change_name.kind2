Jonathan.Joestar #JOJO : U60
Jonathan.Joestar = 42