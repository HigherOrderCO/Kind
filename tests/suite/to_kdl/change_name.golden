// Jonathan.Joestar #JOJO : U60
fun (JOJO) {
  (JOJO) = #42
}

